.github/dependabot.yml

```diff
@@ -9,3 +9,7 @@ updates:
     directory: "/src/SourceBrowser/" # Location of package manifests
     schedule:
       interval: "weekly"
+    groups:
+      microsoft-build-deps:
+        patterns:
+          - "Microsoft.Build.*"
```

Comment:
Is this going to match `Microsoft.Build` (no trailing dot)?

https://github.com/dotnet/source-indexer/blob/e0b40eee9fda1f439c9a4f854012a45f881764e4/src/SourceBrowser/src/Directory.Packages.props#L13